spec/index.md

```diff
@@ -427,6 +427,10 @@ The state tree contains information about the topology of the Internet Computer.
         principal = bytes .size (0..29)
         tagged<t> = #6.55799(t) ; the CBOR tag
 
+-   `/subnet/<subnet_id>/metrics` (blob)
```

Comment:
This is not fully fleshed out. We could also imagine an alternative of a separate path per metric. There's probably a trade-off here between having a single struct with all metrics and how difficult it is to verify the certificate for a single blob or having to perform verification for every different path listed.